src/sentry/options/defaults.py

```diff
@@ -163,6 +163,10 @@
 # From 0.0 to 1.0: Randomly disable normalization code in interfaces when loading from db
 register("store.empty-interface-sample-rate", default=0.0)
 
+# Enable multiple topics for eventstream. It allows specific event types to be sent
+# to specific topic.
+register("store.eventstream-per-type-topic", default=False)
```

Comment:
Add `flags=FLAG_PRIORITIZE_DISK`.

(We frankly should have this on most if not all of these, I'm not sure why people stopped.)